x/evm/keeper/vote_handler.go

```diff
@@ -53,20 +53,34 @@ func (v voteHandler) HandleExpiredPoll(ctx sdk.Context, poll vote.Poll) error {
 		return fmt.Errorf("reward pool not set for poll %s", poll.GetID().String())
 	}
 
-	// TODO: MarkMissingVote for those who didn't vote in time. Need
-	// to be able to get chain of expired polls in order to do it.
+	md := mustGetMetadata(poll)
 	rewardPool := v.rewarder.GetPool(ctx, rewardPoolName)
+	chain, ok := v.nexus.GetChain(ctx, md.Chain)
```

Comment:
is it possible chain not found here?

Reply:
don't think so, but since other places like https://github.com/axelarnetwork/axelar-core/blob/8cfa7a99cbff0d05f94d787390ffc75bed5fa2c6/x/evm/keeper/vote_handler.go#L173 are doing the same, I decided not to change it for the context of this PR.